plugins/volume/org.xen.xapi.storage.zfs-vol/sr.py

```diff
@@ -99,18 +99,23 @@ def destroy(self, dbg, sr):
     def attach(self, dbg, configuration):
         log.debug('{}: SR.attach: config={}'.format(dbg, configuration))
 
-        # ZFS automagically attaches a pool to a mountpoint on
-        # create/boot/etc, so we basically do nothing here but find
-        # this mountpoint
-
-        # FIXME: study how pools are mounted on boot and see if we
-        # could do real a/detach
+        pool_name = configuration["zpool"]
+        try:
```

Comment:
This "try" may be moved into the pool_import maybe. This will allow use to also call pool_import to ensure that a zpool is mounted. @ydirson what do you think? Should I do it or do you take this PR directly in #17 ?

Reply:
> Should I move this "try" into the pool_import? This will allow use to also call pool_import to ensure that a zpool is mounted.

I'd rather keep in `pool_import` itself just the zfs glue, but it sure makes sense to have a dedicated helper for it inside `sr.py`.

> Should I do it or do you take this PR directly in https://github.com/xcp-ng/xcp-ng-xapi-storage/pull/17 ?

Let me merge this into #17 and we'll wok there